packages/common/src/utils/genres.ts

```diff
@@ -85,6 +85,7 @@ export const getCanonicalName = (genre: Genre | any) => {
   return genre
 }
 
+/** User-facing genre labels. Use `convertGenreLabelToValue` to get the correct genre value (to set as the genre in track metadata). */
```

Comment:
Wow, crazy!

Reply:
Yeah i feel like this is very error prone

Reply:
Agreed. Assuming we just need a fix for now, but wondering if there's a more foolproof way to fix going forward...